docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,91 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
 
 ---
 
-## Getting Started
-### Installing the Scalable Pixel Streaming Frontend   
+## Scalable Pixel Streaming Frontend Pieces
+### The Library
+The Library is the part of the Scalable Pixel Streaming Frontend that consumes the Epic Games Pixel Streaming Frontend and UI Frontend. It includes all of the custom signalling server logic that Scalable Pixel Streaming signalling servers require to work. The Library can be either obtained through [Github](https://github.com/ScalablePixelStreaming/Frontend) or [NPM](https://www.npmjs.com/package/@tensorworks/libspsfrontend). To make use of the Library it must be initialised via HTML and Javascript. The Example below is but one example of initialize the Library.  
+
+### The Typescript Example
+The Typescript Example is a simple HTML, CSS and TypeScript implementation of what a user could create to initialize the Scalable Pixel Streaming Frontend Library. Its role is to instantiate the library's components and help start the Scalable Pixel Streaming connection to the signalling server. Please note  how a user chooses to initialize Library is open to interpretation and the users requirements and is not limited to the Typescript Example.
+## Getting Started; installation and consumption
+### Building for Development vs Production
+Within the Scalable Pixel Streaming Frontend there are several npm scripts that can be used to build the Library and Example for either development or production use cases. When building for development this will enable source maps for debugging. When building for production this will disable source maps, minimize console output and minimize the dist contents. Below are a list of npm scripts for both the Library and Example and what each command does:
```

Comment:
> Within the Scalable Pixel Streaming Frontend there are several npm scripts...

to

> The Scalable Pixel Streaming Frontend packages have several npm scripts...

---

>  either development or production use cases

to

>  either development or production

---

> When building for production this will disable source maps, minimize console output and minimize the dist contents

to

> When building for production this will disable source maps, reduce console output and minify the distributed JavaScript files.

Reply:
resolved in b015fe9e45be6e7fe7028ce4bbd3d972cade4fdf